nix/package.nix

```
@@ -339,10 +287,7 @@ in
       homepage = "https://zed.dev";
       changelog = "https://github.com/zed-industries/zed/releases/tag/v${version}";
```

```suggestion
      homepage = "https://github.com/zedless-editor/zed";
```
Don't care about changelog links